From 06af3ca4f2727cbd7118611366b712b9ef57610e Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Tue, 19 Mar 2013 08:08:42 -0400
Subject: Disable unsupported emulated network devices

Bugzilla: 921974

Disable following network devices as they are not supported:
- ne2k_isa
- ne2k_pci
- pcnet

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

Conflicts:
	default-configs/pci.mak

diff --git a/default-configs/i386-softmmu.mak b/default-configs/i386-softmmu.mak
index eec14d2..52c0e6a 100644
--- a/default-configs/i386-softmmu.mak
+++ b/default-configs/i386-softmmu.mak
@@ -18,7 +18,6 @@ CONFIG_APM=y
 CONFIG_DMA=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
-CONFIG_NE2000_ISA=y
 CONFIG_PIIX_PCI=y
 CONFIG_SOUND=y
 CONFIG_HPET=y
diff --git a/default-configs/mips-softmmu.mak b/default-configs/mips-softmmu.mak
index 3abef29..b72dfee 100644
--- a/default-configs/mips-softmmu.mak
+++ b/default-configs/mips-softmmu.mak
@@ -21,7 +21,6 @@ CONFIG_DMA=y
 CONFIG_PIIX4=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_RC4030=y
 CONFIG_DP8393X=y
diff --git a/default-configs/mips64-softmmu.mak b/default-configs/mips64-softmmu.mak
index 9e67da4..5cbed08 100644
--- a/default-configs/mips64-softmmu.mak
+++ b/default-configs/mips64-softmmu.mak
@@ -21,7 +21,6 @@ CONFIG_DMA=y
 CONFIG_PIIX4=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_RC4030=y
 CONFIG_DP8393X=y
diff --git a/default-configs/mips64el-softmmu.mak b/default-configs/mips64el-softmmu.mak
index 123cfea..2bdad44 100644
--- a/default-configs/mips64el-softmmu.mak
+++ b/default-configs/mips64el-softmmu.mak
@@ -22,7 +22,6 @@ CONFIG_PIIX4=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
 CONFIG_IDE_VIA=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_RC4030=y
 CONFIG_DP8393X=y
diff --git a/default-configs/mipsel-softmmu.mak b/default-configs/mipsel-softmmu.mak
index a0c609e..7dca366 100644
--- a/default-configs/mipsel-softmmu.mak
+++ b/default-configs/mipsel-softmmu.mak
@@ -21,7 +21,6 @@ CONFIG_DMA=y
 CONFIG_PIIX4=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_RC4030=y
 CONFIG_DP8393X=y
diff --git a/default-configs/pci.mak b/default-configs/pci.mak
index 2772224..389ff16 100644
--- a/default-configs/pci.mak
+++ b/default-configs/pci.mak
@@ -4,10 +4,7 @@ CONFIG_VIRTIO=y
 CONFIG_USB_UHCI=y
 CONFIG_USB_EHCI=y
 CONFIG_USB_XHCI=y
-CONFIG_NE2000_PCI=y
 CONFIG_EEPRO100_PCI=y
-CONFIG_PCNET_PCI=y
-CONFIG_PCNET_COMMON=y
 CONFIG_RTL8139_PCI=y
 CONFIG_E1000_PCI=y
 CONFIG_IDE_CORE=y
diff --git a/default-configs/ppc-softmmu.mak b/default-configs/ppc-softmmu.mak
index f9f8a81..12ec96d 100644
--- a/default-configs/ppc-softmmu.mak
+++ b/default-configs/ppc-softmmu.mak
@@ -33,7 +33,6 @@ CONFIG_PPCE500_PCI=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_CMD646=y
 CONFIG_IDE_MACIO=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_PFLASH_CFI01=y
 CONFIG_PFLASH_CFI02=y
diff --git a/default-configs/ppc64-softmmu.mak b/default-configs/ppc64-softmmu.mak
index dc44294..34b4bb3 100644
--- a/default-configs/ppc64-softmmu.mak
+++ b/default-configs/ppc64-softmmu.mak
@@ -33,7 +33,6 @@ CONFIG_PPCE500_PCI=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_CMD646=y
 CONFIG_IDE_MACIO=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_PFLASH_CFI01=y
 CONFIG_PFLASH_CFI02=y
diff --git a/default-configs/ppcemb-softmmu.mak b/default-configs/ppcemb-softmmu.mak
index 1c6bcf9..78cb1a7 100644
--- a/default-configs/ppcemb-softmmu.mak
+++ b/default-configs/ppcemb-softmmu.mak
@@ -28,7 +28,6 @@ CONFIG_PPCE500_PCI=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_CMD646=y
 CONFIG_IDE_MACIO=y
-CONFIG_NE2000_ISA=y
 CONFIG_SOUND=y
 CONFIG_PFLASH_CFI01=y
 CONFIG_PFLASH_CFI02=y
diff --git a/default-configs/x86_64-softmmu.mak b/default-configs/x86_64-softmmu.mak
index fea6c62..7c3b203 100644
--- a/default-configs/x86_64-softmmu.mak
+++ b/default-configs/x86_64-softmmu.mak
@@ -18,7 +18,6 @@ CONFIG_APM=y
 CONFIG_DMA=y
 CONFIG_IDE_ISA=y
 CONFIG_IDE_PIIX=y
-CONFIG_NE2000_ISA=y
 CONFIG_PIIX_PCI=y
 CONFIG_SOUND=y
 CONFIG_HPET=y
